.gitignore

```diff
@@ -31,3 +31,5 @@ node_modules/*
 !node_modules/oae-util
 *.log
 doc_data.txt
+
+node_modules/oae-authentication/.DS_Store
```

Comment:
No need to add each one explicitly. The top line should match all .DS_Store files